src/Microsoft.DotNet.Interactive.FSharp.Tests/ApiTests.fs

```diff
@@ -3,35 +3,39 @@
 
 namespace Microsoft.DotNet.Interactive.FSharp.Tests
 
-open Microsoft.DotNet.Interactive.FSharp.FSharpPocketViewTags
+open Microsoft.DotNet.Interactive.FSharp.FSharpKernelHelpers.Html
 open Xunit
 
 type ApiTests() =
 
     [<Fact>]
     member __.``empty tag``() =
-        Assert.Equal("<div></div>", div.ToString())
+        Assert.Equal("<div></div>", (div [] []).ToString())
 
     [<Fact>]
     member __.``indexer as attribute``() =
-        Assert.Equal("<div class=\"c\"></div>", div.["class", "c"].ToString());
+        Assert.Equal("<div class=\"c\"></div>", (div [_class "c"] []).ToString());
 
     [<Fact>]
-    member __.``inner HTML from content``() =
-        Assert.Equal("<div>d</div>", div.innerHTML("d").ToString())
+    member __.``inner HTML from string``() =
+        Assert.Equal("<div>d</div>", (div [] [str "d"]).ToString())
+
+    [<Fact>]
+    member __.``inner HTML from object``() =
+        Assert.Equal("<div>11</div>", (div [] [object 11]).ToString())
```

Comment:
A more interesting example here would be an object of a type that leads to additional formatting.

Reply:
I've added quite a number of more tests here